src/notebooks_service.py

```diff
@@ -133,8 +133,8 @@ def authenticated(f):
     def decorated(*args, **kwargs):
         token = request.cookies.get(
             auth.cookie_name
-        ) or request.headers.get('Authorization',
-                                 'token').split('token', 1)[1].strip()
+        ) or request.headers.get('Authorization', 'token').split('token',
+                                                                 1)[1].strip()
```

Comment:
Will fail on headers like `Authorization: ehskdfbklshfuiw` with http 500. We can merge and create an issue just after.